core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1153,8 +1153,8 @@ class HttpGoogleServicesDAO(
   }
 
   private def getCloudResourceManagerWithCloudResourceManagerServiceAccount = {
-    val cloudResourceManagerServiceAccountCredential = getCloudResourceManagerServiceAccountCredential
-    val cloudResourceManager = getCloudResourceManager(cloudResourceManagerServiceAccountCredential)
+    val billingServiceAccountCredential = getBillingServiceAccountCredential
+    val cloudResourceManager = getCloudResourceManager(billingServiceAccountCredential)
```

Comment:
the billing SA is an owner on both gpalloc and buffer projects, but not the rawls SA.